src/gluonts/nursery/ts_embeddings/embed_model.py

```diff
@@ -240,7 +256,11 @@ def training_step(self, batch, batch_idx):
             prog_bar=False,
             logger=True,
         )
-        return loss
+        # the loss can become NaN if we pick windows that only contain NaN values. We'll ignore these.
+        if torch.any(loss.isnan()):
```

Comment:
I'm not sure if this is a good idea. Could you take a look at this and advice @vafl ?